source/locale/pt_BR/LC_MESSAGES/architecture.po

```diff
@@ -512,41 +509,40 @@ msgstr "Alguns outros componentes são utilizados na dojot, são eles:"
 
 #: ../../source/architecture.rst:217
 msgid ""
-"postgres: this database is used to persist data from many components, "
-"such as Device Manager."
+"postgres: this database is used to persist data from many components, such "
+"as Device Manager."
 msgstr ""
 "postgres: esse banco de dados é utilizado para persistir informações de "
-"vários componenentes, como do gerenciador de dispositivos."
+"vários componentes, como do gerenciador de dispositivos."
 
 #: ../../source/architecture.rst:220
 msgid ""
-"redis: in-memory database used as cache in many components, such as "
-"service orchestrator, subscription manager, IoT agents, and so on. It is "
-"very light and easy to use."
+"redis: in-memory database used as cache in many components, such as service "
+"orchestrator, subscription manager, IoT agents, and so on. It is very light "
+"and easy to use."
 msgstr ""
 "redis: é um banco de dados em memória usado como cache em vários "
-"componentes, como o serviço de osquestração, gerenciador de subscrição, "
+"componentes, como o serviço de orquestração, gerenciador de subscrição, "
 "agentes IoT e outros. É bem leve e fácil de usar."
 
 #: ../../source/architecture.rst:224
 msgid ""
-"rabbitMQ: message broker used in service orchestrator in order to "
-"implement action flows related that should be applied to messages "
-"received from components."
+"rabbitMQ: message broker used in service orchestrator in order to implement "
+"action flows related that should be applied to messages received from "
+"components."
 msgstr ""
-"rabbitMQ: intermediador de mensagens utilizado no serviço de orquestração"
-" para implementar fluxos de ações relacionados que podem ser aplicados a "
-"mensagens recebidas dos componentes."
+"rabbitMQ: *broker* de mensagens usado no orquestrador de serviço para "
+"implementar fluxos de ação relacionados que devem ser aplicados a mensagens "
+"recebidas de componentes."
 
 #: ../../source/architecture.rst:228
 msgid ""
 "mongo database: widely used database solution that is easy to use and "
-"doesn't add a considerable access overhead (where it was employed in "
-"dojot)."
+"doesn't add a considerable access overhead (where it was employed in dojot)."
 msgstr ""
-"mongo: solução de banco de dados amplamente utilizada que "
-"é fácil de usar e não adiciona esforço de acesso considerável (nos locais"
-" onde foi empregado na dojot)."
+"mongo: solução de banco de dados amplamente utilizada, fácil de usar e não "
+"adiciona esforço de acesso considerável (nos locais onde foi empregado na "
```

Comment:
I think that some computational terms like overhead mustn't be translated; it is easier to understand what *overhead* means instead of understanding *esforço de acesso*.